app/controllers/mixins/breadcrumbs_mixin.rb

```diff
@@ -196,5 +201,22 @@ def breadcrumbs_options
     def x_node_right_cell
       @sb[@sb[:active_accord]].presence || x_node
     end
+
+    def breadcrumb_get_session
```

Comment:
The name of the method doesn't reflect what's actually happening in the body. Can you please come up with something more descriptive? Also getters and setters are an antipattern and if you would not have the `breadctumb_` prefix, rubocop would start complaining about it.